### DISCUSSION TITLE
Lecture 01 - NLP

### DISCUSSION THREAD

**Discussion by DivyaniRathod** [seominjoon/kaist-ai605#53] [Q&A] [answered]
![NL](https://user-images.githubusercontent.com/83975185/117698327-3fbc6000-b1e1-11eb-87d7-48bff27b2880.png)
As from above description, NLP means CW or CWW(Computing with words) i.e. Linguistic Language like Fuzzy, is it correct? I attached above image, is it correct ?

**Comment by seominjoon** [comment, marked as answer]
Hi, I am sorry but this is irrelevant to the class, so closing it.

**Reply by DivyaniRathod** [reply]
Thank you for your valuable response. Actually I confused between your video and some theory but now I get it. I apologize about irrelevant question.
